### PR TITLE
Matin/Add DP2P in Payment Method Page

### PR DESCRIPTION
**Pull request by matin-binary**
# Description
   Add DP2P in Payment Method Page

# Type of Change
    New Feature

### CODE REVIEW

File: src/pages/payment-methods/_dp2p.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+                <Localize translate_text="Our support team is always ready to help resolve any disputes." />
+            ),
+        },
+    ]
+    return (
+        <Fragment>
+            <Header as="h2" size="var(--text-size-xl)" align="center" mb="1.2rem" lh="1.25">
+                {localize('Deriv peer-to-peer (DP2P)')}
+            </Header>
+            {is_mobile ? (
```
Comment (mustofa-binary) [inline src/pages/payment-methods/_dp2p.js:168]: we have Show.Mobile component for this, please take a look at show component 🙏

File: src/pages/payment-methods/_dp2p.js [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        button {
+            width: 100%;
+        }
+    }
+`
+
+const ButtonLearnMore = styled(LinkButton)`
+    height: 4rem;
+    @media ${device.mobileL} {
+        margin-bottom: 1.6rem;
```
Comment (mustofa-binary) [inline src/pages/payment-methods/_dp2p.js:101]: you can use px for media query

Comment (matin-binary) [reply src/pages/payment-methods/_dp2p.js:101]: @mustofa-binary what do you mean?

Comment (mustofa-binary) [reply src/pages/payment-methods/_dp2p.js:101]: rem is used for pre-set sizes, normally for sizing inside media query, we will use `px` instead of `rem` 🙏

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**